tools/gfx/d3d12/d3d12-device.cpp

```diff
@@ -1321,13 +1321,21 @@ Result DeviceImpl::createTextureView(
         viewImpl->m_allocator = m_rtvAllocator;
         D3D12_RENDER_TARGET_VIEW_DESC rtvDesc = {};
         rtvDesc.Format = D3DUtil::getMapFormat(desc.format);
-        isArray = desc.subresourceRange.layerCount > 1;
```

Comment:
This was the cause of the error. The isArray from line 1311 is the correct value (from the underlying resource)